src/cr/cube/cube_slice.py

```diff
@@ -300,6 +301,29 @@ def table_name(self):
         table_name = self._cube.labels()[0][self._index]
         return '%s: %s' % (title, table_name)
 
+    def pvals(self, weighted=True, prune=False, hs_dims=None):
+        """Return 2D ndarray with calculated p-vals.
+
+        This function calculates statistically significant results for
+        categorical contingency tables. The values are calculated for 2D tables
+        only.
+
+        :param weighted: Use weighted counts for zscores
+        :param prune: Prune based on unweighted counts
+        :param hs_dims: Include headers and subtotals (as NaN values)
+        :returns: 2 or 3 Dimensional ndarray, representing the p-values for each
+                  cell of the table-like representation of the crunch cube.
+        """
+        stats = self.zscore(weighted=weighted, prune=prune, hs_dims=hs_dims)
+        res = 2 * (1 - norm.cdf(np.abs(stats)))
```

Comment:
This extensive use of the variable name 'res' seems smelly to me. What does it mean? I assume 'result', which is not descriptive and should at least be spelled out if we must use it. It's easy to confuse with 'residual' for one thing. I recommend using a descriptive term meaningful in the current context, like perhaps 'pvals' in this case. Then when you see the `.pvals()` method returning `pvals` it makes more sense and you can trace their development up through the rest of the method.

Reply:
Done